docs/modules/usage/getting-started.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 3
+---
+
+# ⚡Getting Started
+
+## System Requirements
+* Docker version 26.0.0+ or Docker Desktop 4.31.0+
+* You must be using Linux or Mac OS
+  * If you are on Windows, you must use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+## Installation
+The eaisest way to run OpenHands is in Docker. Use `WORKSPACE_BASE` below to
+specify which folder the OpenHands agent should modify.
+
+See the [Getting Started](https://docs.all-hands.dev/modules/usage/getting-started) guide for
+system requirements and more information.
+
+```bash
+WORKSPACE_BASE=$(pwd)/workspace
+
+docker run -it --pull=always \
+    -e SANDBOX_RUNTIME_CONTAINER_IMAGE=ghcr.io/all-hands-ai/runtime:0.9.2-nikolaik \
+    -e SANDBOX_USER_ID=$(id -u) \
+    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
+    -v $WORKSPACE_BASE:/opt/workspace_base \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -p 3000:3000 \
+    --add-host host.docker.internal:host-gateway \
+    --name openhands-app-$(date +%Y%m%d%H%M%S) \
+    ghcr.io/all-hands-ai/openhands:0.9
+```
```

Comment:
I guess we went back to duplicating the docker command? :D 
It's fine. Just got to make sure both get updated.

Reply:
Yup :(